src/app/views/Layout/Topbar.vue

```diff
@@ -186,10 +186,18 @@ export default class STopbar extends Vue {
     }
   }
 
+  private async updateViewingMode () {
+    const result = await this.$apollo.query({ query: require('@graphql/queries/viewer.gql') }).catch(() => ({ data: {} }))
+    if (result && result.data && result.data.viewer) {
```

Comment:
```suggestion
    if (result?.data?.viewer) {
```

It's not important, but you can use [this syntax](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining) now (even if the Vetur is not yet up to date with this version of TS)

Reply:
Will keep this in mind. This is cool syntax IMO. Thanks for letting me know :)